### PR TITLE
Update Chromatik libraries to latest 1.0.1

### PR DESCRIPTION
**Pull request by jkbelcher**

### CODE REVIEW

**Review by zranger1 — APPROVED** [review]
Everything tested is working - haven't seen this break any of our stuff yet.  I also think that this Chromatik feels generally more refined than the previous version..  New features are great, although I have no idea what I agreed to when I clicked "Agree to Terms". (It wouldn't show me the ToS).

(Rendering is visibily different, but just a little.  It'll be way better in the long run, but may cause us to make brightness adjustments in some patterns.   The "feather" and "sparkle" features are interesting and appreciated steps towards better previs.)

**Comment by andrewlook** [comment]
@zranger1 FWIW I did see a link to terms:

image

here's the link: https://chromatik.co/license/

**Review by andrewlook — DISMISSED** [review]
Everything is working for me when I test locally!

I notice a lot of new stuff in the LXP when I save - mainly some new fields, and a few things have been renamed. Are there API docs/release notes anywhere about what has changed in this latest release?

BTW one minor thing, not a blocker by any means but I notice a maven checksum warning, presumably because we're installing these dependencies locally:

```
Downloading from project-repo: file:///Users/al/code/LXStudio-TE/repo/com/heronarts/glxstudio/1.0.1.TE.1-SNAPSHOT/glxstudio-1.0.1.TE.1-SNAPSHOT.jar
Downloading from project-repo: file:///Users/al/code/LXStudio-TE/repo/com/heronarts/glx/1.0.1.TE.1-SNAPSHOT/glx-1.0.1.TE.1-SNAPSHOT.jar
[WARNING] Could not validate integrity of download from file:///Users/al/code/LXStudio-TE/repo/com/heronarts/glxstudio/1.0.1.TE.1-SNAPSHOT/glxstudio-1.0.1.TE.1-SNAPSHOT.jar
org.eclipse.aether.transfer.ChecksumFailureException: Checksum validation failed, no checksums available
    at org.eclipse.aether.internal.impl.AbstractChecksumPolicy.onNoMoreChecksums (AbstractChecksumPolicy.java:63)
    at org.eclipse.aether.connector.basic.ChecksumValidator.validate (ChecksumValidator.java:112)
    at org.eclipse.aether.connector.basic.BasicRepositoryConnector$GetTaskRunner.runTask (BasicRepositoryConnector.java:484)
    at org.eclipse.aether.connector.basic.BasicRepositoryConnector$TaskRunner.run (BasicRepositoryConnector.java:414)
 (RunnableErrorForwarder.java:66)ncurrency.RunnableErrorForwarder.lambda$wrap$0Progress (2): 1.0 MB | 1.1/1.2 MB
    at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1144)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:642)
    at java.lang.Thread.run (Thread.java:1583)
[WARNING] Checksum validation failed, no checksums available from project-repo for file:///Users/al/code/LXStudio-TE/repo/com/heronarts/glxstudio/1.0.1.TE.1-SNAPSHOT/glxstudio-1.0.1.TE.1-SNAPSHOT.jar
Downloaded from project-repo: file:///Users/al/code/LXStudio-TE/repo/com/heronarts/glxstudio/1.0.1.TE.1-SNAPSHOT/glxstudio-1.0.1.TE.1-SNAPSHOT.jar (1.0 MB at 73 MB/s)
[WARNING] Could not validate integrity of download from file:///Users/al/code/LXStudio-TE/repo/com/heronarts/glx/1.0.1.TE.1-SNAPSHOT/glx-1.0.1.TE.1-SNAPSHOT.jar
org.eclipse.aether.transfer.ChecksumFailureException: Checksum validation failed, no checksums available
    at org.eclipse.aether.internal.impl.AbstractChecksumPolicy.onNoMoreChecksums (AbstractChecksumPolicy.java:63)
    at org.eclipse.aether.connector.basic.ChecksumValidator.validate (ChecksumValidator.java:112)
    at org.eclipse.aether.connector.basic.BasicRepositoryConnector$GetTaskRunner.runTask (BasicRepositoryConnector.java:484)
    at org.eclipse.aether.connector.basic.BasicRepositoryConnector$TaskRunner.run (BasicRepositoryConnector.java:414)
    at org.eclipse.aether.util.concurrency.RunnableErrorForwarder.lambda$wrap$0 (RunnableErrorForwarder.java:66)
    at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1144)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:642)
    at java.lang.Thread.run (Thread.java:1583)
```

**Comment by jkbelcher** [comment]
@andrewlook What maven command are you running that produces these warnings?

File: LXStudio-TE.iml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (andrewlook) [inline LXStudio-TE.iml:1]: oops, removing this `.iml` file since i just upgraded intellij

**Review by andrewlook — APPROVED** [review]
👍 LGTM

**Pull request merged**